src/hooks/useSyntheticsTest.ts

```diff
@@ -56,42 +56,45 @@ export function useSyntheticsTest(steps: RecorderSteps): ITestContext {
 
   const onTest = useCallback(
     async function () {
-      const code = await getCodeFromActions(ipc, steps, 'inline');
+      const code = await getCodeFromActions(electronAPI, steps, 'inline');
       if (!isTestInProgress) {
         // destroy stale state
         dispatch({ data: undefined, event: 'override' });
         const onTestEvent = (_event: IpcRendererEvent, data: TestEvent) => {
           dispatch(data);
         };
 
-        ipc.on('test-event', onTestEvent);
+        // electronAPI.on('test-event', onTestEvent);
```

Comment:
```suggestion
```

We want this removed now? I am assuming it's ok since the tests are passing, will confirm when I smoke test.

Reply:
Oops, yes this can be removed here and the [listener](https://github.com/elastic/synthetics-recorder/pull/376/files#diff-17fc44a47be3f3309d4ce97948f68135a3ca438711a75d645a0e3710aa92098bR67) will be callled within `electron.runTest`. Good spot!!